### PR TITLE
FEATURE: Webhook for post approval events

### PR DESCRIPTION
**Pull request by vinothkannans**

### CODE REVIEW

File: app/models/web_hook_event_type.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ class WebHookEventType < ActiveRecord::Base
   CATEGORY = 5
   TAG = 6
   FLAG = 7
+  APPROVAL = 8
```
Comment (tgxworld) [inline app/models/web_hook_event_type.rb]: Hmm this feel a little too vague for me. What approval are we referring to?

**Comment by tgxworld** [comment]
@vinothkannans We need to update the admin pages as well to list the web hooks that are available.

**Comment by vinothkannans** [comment]
@tgxworld I did the changes requested. The webhook admin page will get updated automatically with new event type :wink:.

**Comment by tgxworld** [comment]
Looks good to me :+1:

**Pull request merged**